ba/org.osate.utils/src/org/osate/utils/FileUtils.java

```diff
@@ -140,12 +140,13 @@ public static void copyFiles(List<File> srcFiles, File destFolder, List<String>
 		}
 	}
 
-	public static void copyIntoFile(File inFile, String string2write) throws IOException {
-		BufferedWriter out;
-		out = new BufferedWriter(new FileWriter(inFile));
-		out.write(string2write);
-		out.newLine();
-		out.close();
+	public static void copyIntoFile(File inFile, String string2write) {
+		try (BufferedWriter out = new BufferedWriter(new FileWriter(inFile))) {
+			out.write(string2write);
+			out.newLine();
+		} catch (IOException e) {
+			e.printStackTrace();
```

Comment:
The original method method propagated any `IOException`s. This version should do the same. Keep the try-with-resources, but don't catch the `IOException` and let it be propagated to the caller.